src/test/java/org/opentripplanner/apis/gtfs/mapping/CarsAllowedMapperTest.java

```diff
@@ -0,0 +1,21 @@
+package org.opentripplanner.apis.gtfs.mapping;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.transit.model.network.CarAccess;
+
+class CarsAllowedMapperTest {
+
+  @Test
+  void mapping() {
+    Arrays
+      .stream(CarAccess.values())
+      .filter(ba -> ba != CarAccess.UNKNOWN)
+      .forEach(d -> {
+        var mapped = CarsAllowedMapper.map(d);
+        assertEquals(d.toString(), mapped.toString());
+      });
```

Comment:
Since the mapper only operates on three cases I think this test would be more complete and more readable if you just added an assertion for each case.

Reply:
I added assertions for each case separately. I also did the same to BikesAllowedMapperTest